tests/src/buildApplication/processVercelOutput.test.ts

```diff
@@ -407,3 +416,72 @@ describe('processVercelOutput', () => {
 		expect(processed).toEqual(expected);
 	});
 });
+
+describe('setupOutputDir', () => {
+	beforeEach(() => {
+		mockFs({
+			'.vercel': {
+				output: {
+					static: {
+						_next: { __private: { trace: 'private trace file' } },
+						'index.js': 'console.log("hello world")',
+						nested: { 'index.html': '<html>Hello world</html>' },
+					},
+				},
+			},
+			existing: { 'file.txt': 'hello world' },
+		});
+	});
+
+	afterEach(() => mockFs.restore());
+
+	const vercelDir = resolve('.vercel', 'output', 'static');
+
+	test('default vercel output dir deletes telemetry file only', async () => {
+		const staticAssets = await getVercelStaticAssets();
```

Comment:
ideally I don't think that in these tests we should use some production code (basically here you are kind of testing both `getVercelStaticAssets` and `setupOutputDir`)

it's not a huge deal as I think we have much worse coupling in other tests but here it should be pretty simple to remove this